warehouse/query-core/src/main/java/datawave/query/util/sortedset/ByteArrayComparator.java

```diff
@@ -1,27 +1,13 @@
 package datawave.query.util.sortedset;
 
+import org.apache.hadoop.io.WritableComparator;
+
 import java.util.Comparator;
 
 public class ByteArrayComparator implements Comparator<byte[]> {
 
     @Override
     public int compare(byte[] data, byte[] term) {
-        int minSize = data.length;
-        if (term.length < minSize)
-            minSize = term.length;
-        int comparison = 0;
-        for (int i = 0; i < minSize; i++) {
-            comparison = new Byte(data[i]).compareTo(term[i]);
-            if (comparison != 0)
-                break;
-        }
-        if (comparison == 0) {
-            if (minSize < data.length) {
-                comparison = 1;
-            } else if (minSize < term.length) {
-                comparison = -1;
-            }
-        }
-        return comparison;
+        return WritableComparator.compareBytes(data, 0, data.length, term, 0, term.length);
```

Comment:
No test?

Reply:
I know... figured this out last thing last night.... I will put something in